### ISSUE TITLE
Problem with OnMessage (Newtonsoft.Json.Linq.JObject values, ...)

### ISSUE THREAD

**Issue by MKahmen** [rdelrosario/xamarin-plugins#33]
Hi,

```
public async void OnMessage (Newtonsoft.Json.Linq.JObject values, PushNotification.Plugin.Abstractions.DeviceType deviceType)
        {
            if (deviceType == DeviceType.iOS) {
                Debug.WriteLine (values.SelectToken("aps"));
        }
```

Gives me the following output:

```
{
    alert = "This is a test alert.";
    badge = 0;
    sound = default;
}
```

I want to parse `alert` but it seems like I can't since the `"` are missing around it, right?
How can I retrieve the string `This is a test alert.`?
Any tipps / ideas?

BR

**Comment by rdelrosario** [comment]
This is Android or iOS?

**Comment by MKahmen** [comment]
iOS. Push sent with PushSharp.

**Comment by charri** [comment]
Fixed with #37

**Comment by rdelrosario** [comment]
Thanks for the fix @charri. Will release a new NuGet package version with this fix.
